Automattic/wp-calypso
# Signup: Add a log in link to the 2FA error

This pull request adds a log in link to the error we show when users try to connect their Google account to a WordPress.com account with 2FA enabled.
  
#### Testing instructions
  
1. Run `git checkout update/social-signup-2FA` and start your server
2. Apply D7200 to your sanbox
2. Open the [`Signup` page](http://calypso.localhost:3000/start)
3. Go through the signup flow
4. Connect with Google on the user step and select an account which has the same email address as a WordPress.com account with 2FA enabled.
5. Check that you see an error message and that there is a link to log in
6. Check that the log in flow redirects you back to signup

#### Additional notes
Ideally, the user should be able to log in as part of the signup flow. We'll consider this at a later point.
  
#### Reviews
